atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -104,8 +120,8 @@ def run(self):
 
             os.chmod(config_path, 0o444)
 
-        # Set build args
-        self.workflow.builder.buildargs.update(self.buildargs)
```

Comment:
I think we want to do both, keep the buildargs for BW compatibility and also generate the file.

Reply:
Makes sense to leave there both. @rcerven @MartinBasti what's your thoughts on this?

Reply:
yes both

Reply:
Ok returned